docs/src/modules/components/MarkdownElement.js

```diff
@@ -253,23 +253,20 @@ const Root = styled('div')(({ theme }) => ({
     flexShrink: 0,
     backgroundColor: theme.palette.divider,
   },
-  '& kbd.key': {
-    // Style taken from GitHub
-    padding: '4px 5px',
+  '& kbd': {
```

Comment:
This is a regression: 
<img width="74" alt="Screenshot 2022-04-07 at 13 15 42" src="https://user-images.githubusercontent.com/3165635/162186836-8a35f4a5-ef87-43eb-83db-5ae9ccfb9d35.png">


Please blame the line to understand the history, and why it needs to be done this way.

```suggestion
  '& kbd.key': {
```